### ISSUE TITLE
[Bug]: Screen Reader is announcing MessageBar title Component multiple times

### ISSUE THREAD

**Issue by Vaishnavi0609** [microsoft/fluentui#33540]
### Component

MessageBar

### Package version

9.0.25

### React version

17.0.2

### Environment

```shell
System:
    OS: Linux 5.0 undefined
    CPU: (8) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 0 Bytes / 0 Bytes
    Shell: 1.0 - /bin/jsh
  npmPackages:
    @fluentui/react: 8.112.3 => 8.112.3 
    @fluentui/react-components: 9.47.5 => 9.47.5 
    @types/react: ^18.3.18 => 18.3.18 
    @types/react-dom: ^18.3.5 => 18.3.5 
    react: ^18.3.1 => 18.3.1 
    react-dom: ^18.3.1 => 18.3.1
```

### Current Behavior

Screen Reader (Narrator and NVDA) reads MessageBarTitle component multiple times

### Expected Behavior

Screen Reader (Narrator and NVDA) should read MessageBarTitle component once

### Reproduction

https://vitejsvite9lw7yakt-jtzf--5173--c8c182a3.local-credentialless.webcontainer.io/

### Steps to reproduce

1. open the sandbox link
2. switch on the Screen Reader (Eg. Narrator)
3. The screen reader reads the MessageBarTiltle multiple times

### Are you reporting an Accessibility issue?

None

### Suggested severity

Medium - Has workaround

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by Hotell** [comment]
@Vaishnavi0609 the repro link doesn't work https://vitejsvite9lw7yakt-jtzf--5173--c8c182a3.local-credentialless.webcontainer.io/ can you provide a working one ? ty

**Comment by Vaishnavi0609** [comment]
Hi @Hotell , Sorry , can you please try the link https://stackblitz.com/edit/vitejs-vite-9lw7yakt?file=src%2FApp.tsx
